### ISSUE TITLE
Bug report: Lock screen lets keystrokes pass through to the active pane below if the password-field does not have keyboard-focus

### ISSUE THREAD

**Issue by Gewerd-Strauss** [ericbiewener/obsidian-lock-screen-plugin#11]
Hello,

whenever the screen of the application is locked and the input field does not have keyboard focus, all send keystrokes are passed-through to the active pane below. This can regularly happen when obsidian is launched, but the user tabs out of obsidian before the plugin has loaded and the lockscreen appeared. In this case, any keys send are not captured by the password field, and instead reappear at line 1, column 1 of whatever file is active in obsidian when it starts. Additionally this can also happen over the course of a day, but I could not narrow down what conditions must be met for this focus-loss to occur in this case. For the startup-issue, this occurs 100% of the time when I try to replicate it. 

I would like for this to get fixed. I know it is not exactly a gigantic bug, but a regular inconvenience. In my case, this breaks the yaml-header of my dailynote. Said note takes a about 20 seconds to load because it requires about 15 dv-queries. Usually it can preload and render either completely or partially between when the program autostarting and me typing in my password, but this bug regularly renders that pointless. The passed-through text  will kill the yaml. I then log on, remove the passed-through string from the yaml, which in turn triggers a new rendering - and thus I sit there for 20 seconds. 

I hope this can be rectified.

Thank you.
Sincerely,
~Gw

**Comment by Gewerd-Strauss** [comment]
Additionally, I have checked that the console does not receive any errormessages during the testing process.

**Comment by grrowl** [comment]
this also happens on mobile when you launch Obsidian via the "share to.." sheet. The pop up "which note to append to" has focus, not the password field
